### PR TITLE
[FIX][website_sale_cart_expire] fix singleton

### LINKED ISSUE #755: [14.0] website_sale_cart_expire
## Module
website_sale_cart_expire

## Describe the bug
when the cron run i obtain that errore:
Traceback (most recent call last):
  File "/home/odoo/src/odoo/odoo/tools/safe_eval.py", line 331, in safe_eval
    return unsafe_eval(c, globals_dict, locals_dict)
  File "", line 1, in <module>
  File "/home/odoo/src/user/b_OCA/e-commerce/website_sale_cart_expire/models/website.py", line 45, in _scheduler_website_expire_cart
    carts_to_expire.action_cancel()
  File "/home/odoo/src/odoo/addons/sale/models/sale.py", line 825, in action_cancel
    'context': {'default_order_id': self.id},
  File "/home/odoo/src/odoo/odoo/fields.py", line 3822, in __get__
    raise ValueError("Expected singleton: %s" % record)
ValueError: Expected singleton: sale.order(34961, 34933, 34927, 34923, 34914, 34893, 34867, 34856, 34830, 34807, 34806, 34782, 34123, 34757, 34753, 34752, 34727, 34714, 34708, 34691, 34690, 34689, 34631, 34613, 34579, 34578, 34576, 34562, 34530, 34507, 34464, 34451, 34438, 34415, 34402, 34395, 34372, 3

## To Reproduce
**Affected versions**:

- [x] 14.0 https://github.com/OCA/e-commerce/pull/756

1. install module
2. not finish some order on website
3. obtain error

**Expected behavior**
there is no  error

**Additional context**
Add any other context about the problem here. (e.g. OS, Python version, ...)

### PR DESCRIPTION
**Pull request by matteoopenf**
Resolve: https://github.com/OCA/e-commerce/issues/755

### CODE REVIEW

File: website_sale_cart_expire/models/website.py [hunk 1]
```diff
@@ -42,4 +42,4 @@ def _scheduler_website_expire_cart(self):
         # Expire carts
         for cart in carts_to_expire:
             cart.message_post(body=_("Cart expired"))
-        carts_to_expire.action_cancel()
+            cart.action_cancel()
```
Comment (chienandalu) [inline website_sale_cart_expire/models/website.py]:
```suggestion
        carts_to_expire._action_cancel()
```

**Review by chienandalu — APPROVED** [review]
Thanks :slightly_smiling_face: :+1:

**Comment by matteoopenf** [comment]
[chienandalu](https://github.com/chienandalu) thank you for advise

**Comment by matteoopenf** [comment]
@ivantodorovich, is it ok for you?

**Review by patrickt-oforce — APPROVED** [review]

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please use proper commit message, and include full details of the problem

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#71commit-message

**Comment by pedrobaeza** [comment]
Still incorrect and with 2 commits I'm afraid.

**Comment by matteoopenf** [comment]
> I'm afraid

i will merge the commit and change the commit message

**Comment by matteoopenf** [comment]
@pedrobaeza  now, is it ok for you?

**Comment by pedrobaeza** [comment]
Well, it's not for me, but for the OCA guidelines. It's still not correct. The name of the module doesn't go between brackets, and still missing the details:

```
[FIX] website_sale_cart_expire: no singleton error in cron

When the cron run, this error is obtained:

Traceback (most recent call last):
File "/home/odoo/src/odoo/odoo/tools/safe_eval.py", line 331, in safe_eval
return unsafe_eval(c, globals_dict, locals_dict)
File "", line 1, in
File "/home/odoo/src/user/b_OCA/e-commerce/website_sale_cart_expire/models/website.py", line 45, in _scheduler_website_expire_cart
carts_to_expire.action_cancel()
File "/home/odoo/src/odoo/addons/sale/models/sale.py", line 825, in action_cancel
'context': {'default_order_id': self.id},
File "/home/odoo/src/odoo/odoo/fields.py", line 3822, in get
raise ValueError("Expected singleton: %s" % record)
ValueError: Expected singleton: sale.order(34961, 34933, 34927, 34923, 34914, 34893, 34867, 34856, 34830, 34807, 34806, ...

Fixed running the inner method that is multi-record aware.
```

**Comment by matteoopenf** [comment]
@pedrobaeza  done and thanks for you advise

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**